app/lib/rocketchat.js

```diff
@@ -169,6 +169,17 @@ const RocketChat = {
 			message: I18n.t('Not_RC_Server', { contact: I18n.t('Contact_your_server_admin') })
 		};
 	},
+	async getServerTimeSync(server) {
+		try {
+			const response = await RNFetchBlob.fetch('GET', `${server}/_timesync`);
```

Comment:
This request takes 1600ms on https://mobile.rocket.chat, so we should be careful on the impact.
If screen lock is enabled, we do it right on app open.
Pay attention on the white screen after the splash screen is hidden.

https://user-images.githubusercontent.com/804994/153067384-32401176-9f43-43ee-8ec0-500453ea54bf.mp4

https://user-images.githubusercontent.com/804994/153068220-e3d8a199-887d-473a-a964-60ce661041f8.mp4

A good solution for this is to move `RNBootSplash.hide()` to after the call.